test/e2e/singlecluster/e2e_test.go

```diff
@@ -122,12 +122,15 @@ var _ = ginkgo.Describe("Kueue visibility server", func() {
 					Queue(localQueueA.Name).
 					Image("gcr.io/k8s-staging-perf-tests/sleep:v0.0.3", []string{"60s", "-termination-grace-period", "0s"}).
 					Request(corev1.ResourceCPU, "1").
-					TerimnationGracePeriod(1).
+					TerminationGracePeriod(1).
 					BackoffLimit(0).
 					PriorityClass(highPriorityClass.Name).
 					Obj()
 				gomega.Expect(k8sClient.Create(ctx, blockingJob)).Should(gomega.Succeed())
 			})
+			ginkgo.By("Ensure the workload is admitted, by awaiting until the job is unsuspended", func() {
+				expectJobUnsuspended(client.ObjectKeyFromObject(blockingJob))
+			})
```

Comment:
Should we use `ExpectWorkloadsToBeAdmitted()`?

https://github.com/kubernetes-sigs/kueue/blob/5fced9e6f0b1323153d2df65816e70a087c8f6f7/test/util/util.go#L268

Reply:
In e2e tests we tend to use the API that is used by users, whereas Workload is more of a "technical detail API" of Kueue.

Reply:
It makes sense.